### ISSUE TITLE
Reproduce PatchConvnet

### ISSUE THREAD

**Issue by billpsomas** [facebookresearch/deit#185]
Hello,

If I want to reproduce the numbers of the PatchConvnet, what is the command that I have to run? Where can I find it?

Thank you!

**Comment by billpsomas** [comment]
Hello again,

Is this a correct command to train S120?

`python3 -m torch.distributed.launch --nproc_per_node=4 --use_env /iarai/home/bill.psomas/code/deit/main.py --model S120 --batch-size 256 --data-path /path/to/imagenet/ --output_dir /path/to/output/`

Should some hyperparameters change to?

Thank you.

**Comment by TouvronHugo** [comment]
Hi @billpsomas,
Thanks for your question.
You need to change the optimiser (LAMB instead of AdamW), adapt the learning rate (3e-3), the global batch size (2048) and adapt stochastic depth (I refer you to the [paper](https://arxiv.org/pdf/2112.13692.pdf) for the right value stochastic depth depending on the size of the model).

Best,

Hugo
